salt/states/cmd.py

```diff
@@ -114,6 +114,31 @@
    Note that if the ``cmd.wait`` state also specifies ``stateful: True`` it can
    then be watched by some other states as well.
 
+4. :strong:`The stateful argument can optionally include a test_name parameter.`
+
+   This is used to specify a command to run in test mode.  This command should
+   return stateful data for changes that would be made by the command in the
+   name parameter.
+    
+   .. versionadded:: 2015.3.0
```

Comment:
Should be 2015.2.0.  It will be March when the release comes out, but 2015.2.0 is the name we're stuck with. :-)